Documentation/ceph-dashboard.md

```diff
@@ -60,6 +60,7 @@ The following dashboard configuration settings are supported:
       urlPrefix: /ceph-dashboard
       port: 8443
       ssl: true
+      role: read-only
```

Comment:
As I'm interpreting the Ceph doc, seems like we should allow for a list of roles such as:
```
 roles:
 - read-only
 - rbd/pool-manager
```

Reply:
yes, according to ac-user-set-roles:
```
dashboard ac-user-set-roles <username> <roles> [<roles>...] :  Set user roles
```
But since we don't manage users and have only admin IMO I don't see sense in fine tuning admin, since it have full rights eitherway

Reply:
Currently rook only creates a dashboard user named "admin". To really have a complete solution, we could even create other users with different levels of access, but that would be much more complicated than what you want to accomplish. What about this to keep it simple? Then we don't need to worry about configuring other users or exposing other roles in the CR.
```
readOnly: true
```

Reply:
makes sense, I'll adapt PR accordingly

Reply:
@travisn pushed updated version